ftl/core/notetypes.ftl

```diff
@@ -54,3 +54,4 @@ notetype-error-no-image-to-show = No image to show.
 notetypes-no-occlusion-created = You must make at least one occlusion.
 notetypes-no-occlusion-created2 = Unable to add. Either you have not added any occlusions, or the first field is empty.
 notetypes-io-select-image = Select Image
+notetypes-io-select-image-from-clipboard = Select Image from Clipboard
```

Comment:
(minor) Would suggest changing this to `Paste Image from Clipboard` to signify that there is no selection screen.